docs/src/browsers.md

```diff
@@ -430,6 +428,67 @@ set NODE_EXTRA_CA_CERTS="C:\certs\root.crt"
 $env:NODE_EXTRA_CA_CERTS="C:\certs\root.crt"
 ```
 
+If your network is slow to connect to Playwright browser archive, you can increase the connection timeout in milliseconds with `PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT` environment variable:
+
+```bash tab=bash-bash lang=js
+PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=120000 npx playwright install
+```
+
+```batch tab=bash-batch lang=js
+set PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=120000
+npx playwright install
+```
+
+```powershell tab=bash-powershell lang=js
+$env:PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT="120000"
+npx playwright install
+```
+
+```bash tab=bash-bash lang=python
+pip install playwright
+PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=120000 playwright install
+```
+
+```batch tab=bash-batch lang=python
+set PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=120000
+pip install playwright
+playwright install
+```
+
+```powershell tab=bash-powershell lang=python
+$env:PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT="120000"
+pip install playwright
+playwright install
+```
+
+```bash tab=bash-bash lang=java
+PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=120000 mvn test
+```
+
+```batch tab=bash-batch lang=java
+set PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT=120000
+mvn test
```

Comment:
shouldn't it be our favorite`mvn exec:java -e -Dexec.mainClass=com.microsoft.playwright.CLI -Dexec.args="install"` ?

Reply:
Ouch!